grype/db/status.go

```diff
@@ -3,9 +3,9 @@ package db
 import "time"
 
 type Status struct {
-	Age                   time.Time
-	CurrentSchemaVersion  int
-	RequiredSchemaVersion int
-	Location              string
-	Err                   error
+	Age           time.Time `json:"dataAge"`
```

Comment:
Usually "age" is a duration — in seconds, days, years, etc. It looks like this is showing the date (a point in time) that the database was built, and I see it's referred to with the "Built:" field when running `grype db status`. Could we adjust this name?